hypoddpy/hypodd_relocator.py

```diff
@@ -423,7 +423,7 @@ def _read_event_information(self):
         self.log("Reading all events...")
         catalog = Catalog()
         for event in self.event_files:
-            catalog += readEvents(event)
+            catalog += read_events(event).events[0]
```

Comment:
Yes, I will revert this back, I got only one event in the catalog object. Therefore, I forgot this, I will revert it.